### PR TITLE
fix container networking

### PR DESCRIPTION
**Pull request by dyc3**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,6 +11,34 @@ def hello_world():
 def agent_dumb():
 	return Response('{"type": 0, "args": {"target": {"party":0, "slot": 0}, "move": 0}}', mimetype="application/json")
 
+@app.route('/test')
```
Comment (elenirotsides) [inline app.py:14]: @dyc3 Is this supposed to be a GET request to http://localhost:4000/test ?

Comment (dyc3) [reply app.py:14]: yes

**Review by elenirotsides — CHANGES_REQUESTED** [review]
I'm getting a `404 page not found` when I try and make a GET request to http://localhost:4000/test .

**Comment by dyc3** [comment]
Wrong port, its localhost:5000

**Comment by elenirotsides** [comment]
That one gives me a different error, `Error: socket hang up`

**Comment by dyc3** [comment]
Oh yeah, that's because #3 isn't resolved yet. Comment out the `main()` that starts the discord bot and try it

**Review by elenirotsides — APPROVED** [review]

**Pull request merged**